website/content/docs/secrets/pki/scep.mdx

```diff
@@ -0,0 +1,229 @@
+---
+layout: docs
+page_title: Simple Certificate Enrollment Protocol (SCEP)
+description: >-
+   Understand the configuration and limitations of Vault's PKI secrets engine implementation of the Simple Certificate Enrollment Protocol (SCEP).
+---
+
+# Simple Certificate Enrollment Protocol (SCEP) <EnterpriseAlert inline="true" />
+
+This document covers configuration and limitations of Vault's PKI Secrets Engine
+implementation of the [SCEP](https://datatracker.ietf.org/doc/html/rfc8894) <EnterpriseAlert inline="true" />.
+
+## What is Simple Certificate Enrollment Protocol (SCEP)?
+
+SCEP is an IETF standardized protocol, [RFC 8894](https://datatracker.ietf.org/doc/html/rfc8894),
+that allows clients to acquire client certificates and associated
+Certificate Authority (CA) certificates.
+
+## Enabling SCEP support on a Vault PKI mount
+
+The following is a list of steps required to configure an existing PKI
+mount to serve SCEP clients. Each of which can be broken down within three main
+categories.
+
+ 1. [Authentication mechanisms](#configuring-scep-authentication-mounts)
+ 1. [Updating PKI tunable parameters](#updating-the-pki-mount-tunable-parameters)
+ 1. [PKI SCEP configuration](#pki-scep-configuration)
+
+### Configuring SCEP Authentication mounts
+
+The Vault SCEP implementation supports a few different authentication mechanisms in
+two categories.
+
+ 1. Challenge Based authentication
+   2. [Intune authentication](/vault/docs/auth/scep)
```

Comment:
The nested bullet points were not rendered correctly:
<img width="319" alt="image" src="https://github.com/user-attachments/assets/f89f1d1d-824a-4460-8c38-dfeb8acf1253" />